File: your_order_please.js
function order(words){
    let stringArr=words.split(" ").sort(function(a,b){
      return a.match(/\d/)-b.match(/\d/)
    })
    //we can use any type condition along with regex inside our compare function to determine the order in which our data should be sorted
    return stringArr.join(" ")
  }

File: your_order_please.py
def order(sentence):
  # code here
    import re
    string_arr= sentence.split(" ")
    sorted_arr=sorted(string_arr,key=lambda word: re.findall("\d",word))
    print(string_arr, sorted_arr)
    str=" "
    final_result=str.join(sorted_arr)
    print(final_result)
    return final_result